.gitignore

```diff
@@ -1 +1,2 @@
 /target
+.vscode/
```

Comment:
While the whole project is not dependent uniquely on vscode I think such ignores should be in the developer's system wide gitignore